app/controllers/api/databases.php

```diff
@@ -3375,7 +3391,28 @@ function updateAttribute(
             }
         };
```

Comment:
I found a trick for fixing this alignment problem. Would you please try wrapping the function with parenthesis like:

```php
$checkPermissions = (function() {
    // Logic
});
```